src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts

```diff
@@ -121,6 +121,12 @@ export class CephfsSubvolumeListComponent implements OnInit, OnChanges {
             { size: 'lg' }
           )
       },
+      {
```

Comment:
@Pegonzal Why something related to edit subvolume is added? Is this intentional?

Reply:
@avanthakkar that's on me. I accidentally removed the subvolume edit button in one of my PR. @Pegonzal saw that and he is fixing it here!